templates/ai/consulting.html

```diff
@@ -234,30 +234,24 @@ <h3>
   <div class="row">
     <div class="col-12">
       <div class="row p-divider">
-        <div class="col-6 p-divider__block">
-            <div class="p-heading-icon--muted">
-              <div class="p-heading-icon__header">
-                <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}26797d10-newsfeed-stripvideo-icon.svg" width="32" alt="">
-                <h4 class="p-heading-icon__title">Webinars</h4>
-              </div>
+        <div class="col-4 p-divider__block">
+          <div class="p-heading-icon--muted">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}26797d10-newsfeed-stripvideo-icon.svg" width="32" alt="">
+              <h4 class="p-heading-icon__title">Webinars</h4>
             </div>
+          </div>
           <p>
             A detailed look into the artificial intelligence and machine learning world. Learn about the AI landscape, how to deploy your first model, and more.
           </p>
           <p>
-            <a
-              class="p-link--external"
-              href="https://blog.ubuntu.com/2018/07/26/ai-ml-ubuntu-everything-you-need-to-know"
-              >AI, ML, & Ubuntu: Everything you need to know.</a
-            >
+            <a href="/blog/ai-ml-ubuntu-everything-you-need-to-know">AI, ML, & Ubuntu: Everything you need to know.</a>
```

Comment:
I think you need to remove the full stop from this one, I know it was there, but none of the other links have them.